aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -10,9 +10,10 @@ export const ThemeModeToggle = () => {
   return (
     <Box width={{ max: 'small' }}>
       <Button
-        color={{ dark: 'blue!', light: 'white' }}
+        color='background-back'
         label={label}
         icon={icon}
+        plain
```

Comment:
Can we style this the same way as NavButtons instead of using plain? I assumed that on hover it would have a background to allow it to appear more "clickable".

Reply:
awe yes that was before I was using `NavButton`